angr/simos/windows.py

```diff
@@ -303,6 +306,25 @@ def link_list(mods, offset):
             if isinstance(loaded_object, cle.backends.pe.PE):
                 self._init_object_pe_security_cookie(loaded_object, state, kwargs)
 
+        if fs is None:
+            fs = {}
+
+        for name in fs:
+            if type(fs[name]) is str:
+                fs[name] = fs[name].encode('uft-8')
```

Comment:
`utf-8` I suppose.

Also, on Windows, file names are encoded in native encoding, not UTF-8...

Reply:
Sorry about the typo!

What encoding should I use? ansi?

Reply:
`fs[name]` does not hold file content?